external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/JsonUtils.scala

```diff
@@ -95,4 +95,20 @@ private object JsonUtils {
     }
     Serialization.write(result)
   }
+
+  /**
+   * Write per-topic partition lag as json string
+   */
+  def partitionLags(latestOffsets: Map[TopicPartition, Long],
+                    processedOffsets: Map[TopicPartition, Long]): String = {
```

Comment:
nit:

```scala
def partitionLags(
    latestOffsets: Map[TopicPartition, Long],
    processedOffsets: Map[TopicPartition, Long]): String = {
```

per https://github.com/databricks/scala-style-guide#spacing-and-indentation

Please feel free to address it with other comments later.

Reply:
addressed. would it be possible to add this to scala style checks ?

Reply:
Please go ahead if possible.